models/PyCryptoBot.py

```diff
@@ -357,11 +357,13 @@ def getBuyMaxSize(self):
             return None
 
     def getDateFromISO8601Str(self, date: str) :
+		
         new_date_str = f'{date} 00:00:00' if len(date) == 10 else date
-        dt = new_date_str.split('T')
+																   
+        dt = new_date_str.split(' ')
```

Comment:
the original code splits on 'T', new code splits on ' ' (space), we might get the better of both using something like:
```
>>> re.split( r'[bg ]', '123b456g987wer 543' )
['123', '456', '987wer', '543']
```

Reply:
I will update the function to deal with both space and 'T' and remove any .replace parts in the main code.